raddb/mods-config/sql/main/postgresql/queries.conf

```diff
@@ -246,7 +250,11 @@ accounting {
 					NULL, \
 					'%{Service-Type}', \
 					'%{Framed-Protocol}', \
-					NULLIF('%{Framed-IP-Address}', '')::inet)"
+					NULLIF('%{Framed-IP-Address}', '')::inet), \
+					NULLIF('%{Framed-IPv6-Address}', '')::inet), \
+					NULLIF('%{Framed-IPv6-Prefix}', '')::inet), \
+					NULLIF('%{Framed-Interface-Id}', '')::inet), \
+					NULLIF('%{Delegated-IPv6-Prefix}', '')::inet)"
```

Comment:
Ah yes... The deleted line has `...inet)` because it's at the end of the enclosing `(...)` block.  In this patch, the first 4 lines should have `...:inet,` and only the last one has `...:inet)`

Can you make those changes in your branch `git commit --amend` && do a `git push -f` to update this PR?

Thanks.